portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +36,10 @@ function addRandomIntro() {
   // Add it to the page.
   const introContainer = document.getElementById('intro-container');
   introContainer.innerText = intro;
+}
+
+async function getRandomQuoteUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const quote = await response.text();
```

Comment:
Please use let for local values that get overwritten. Constants are generally only used for data that gets assigned once and never changed again. 

let response = ...
let quote = ...

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let